packages/components/src/FormField/FormField.tsx

```diff
@@ -252,6 +259,11 @@ export function FormField({
         {placeholder || " "}
       </label>
       {fieldElement()}
+      {type === "text" && loading && (
+        <span className={styles.icon}>
```

Comment:
Are you ok changing this `icon` classname to `postfix` or something since that's what it's more shifted into here?